src/main/scala/jigg/util/JSONUtil.scala

```diff
@@ -0,0 +1,133 @@
+package jigg.util
+
+import java.io._
+import scala.xml._
+import scala.io.Source
+import scala.collection.mutable.StringBuilder
+
+object JSONUtil {
+
+  object XMLParser {
+    // If text node has newline character, Node.child method doesn't work collectly.
+    def trimNode(x: Node): Node = 
+      XML.loadString(x.toString.split("\n").mkString)
```

Comment:
I don't think this is the correct thing to do. In the command below:

```sh
$ echo "dogs\nrun" | java -cp target/jigg-assembly-0.6.jar:jar/stanford-corenlp-3.6.0-models.jar jigg.pipeline.Pipeline -annotators "corenlp[tokenize,ssplit]" -outputFormat json
```

The result is:
```json
{"root":{ "_":[
...
{ "sentence":{
"text":"dogsrun",
...
{ "token":{
"form":"dogs",
"id":"t0",
"characterOffsetBegin":"0",
"characterOffsetEnd":"4"}},
{ "token":{
"form":"run",
"id":"t1",
"characterOffsetBegin":"5",
"characterOffsetEnd":"8"}}]
...
```
and offsetBegin = 5 for "run" is wrong (we cannot recover the original text from text).

What is the problem when newline character is included in text? Perhaps the problem can be solved with `XMLUtil.text` method? This tries to extract just the original text from the given node.